packages/mdc-tabs/tab/mdc-tab.scss

```diff
@@ -58,18 +58,6 @@ $mdc-tab-with-icon-and-text-height: 72px;
     @include mdc-theme-prop(outline-color, text-secondary-on-light);
```

Comment:
Can you replace all occurrences of `-on-light` with `-on-background` in this file?